### ISSUE TITLE
[Feature Request] Add a user setting for Fork to be a Single Instance application.

### ISSUE THREAD

**Issue by Grahamvs** [ForkIssues/TrackerWin#988]
It would be great if we had an option to make Fork behave as a Single Instance application based on a user setting.

**Comment by felixhirt** [comment]
Just mentioning #266 to make sure you know about the cli features. how do you start the second fork instance right now?

**Comment by Grahamvs** [comment]
Currently I manage to start a second instance on Windows by using the shortcut pinned to the taskbar. I have 2 monitors and generally use Fork on my second screen, and since I've set windows to only show apps on the taskbar if they're on the that screen, I'm able to click the pinned shortcut on the main screen multie times, and it opens a new instance for each click.

Edit:
I'll try out what's being mentioned in the linked issue when I get a chance, but that might not be for a few more weeks.

**Comment by xmedeko** [comment]
I agree with @Grahamvs a user setting is more convenient then a CLI param. Also, clicking on the taskbar/desktop icon should bring focus to the first instance.

**Comment by snicho99** [comment]
Came looking to +1 this feature request. I'm afraid I have the bad habit of hitting Windows Key + typing the first couple of letters of the app I'm looking for. For a huge chunk of windows apps that gives the current instance of whatever you're running, but by the mid morning I tend to have about 10 Fork windows open. ;-)
Such a minor quibble tho - Love the app otherwise by the way.

**Comment by mitchcapper** [comment]
Some of the other tickets mentioning this reference removing multiple instances completely.  It would be great to make sure this is only an option if it gets added.  I frequently use multiple instances each with their own workspaces open.
